Entities/SceneLayer.cpp

```diff
@@ -233,18 +266,49 @@ namespace RTE {
 			return;
 		}
 		_putpixel(m_MainBitmap, pixelX, pixelY, materialID);
+
+		//RegisterDrawing(pixelX, pixelY, pixelX, pixelY);
```

Comment:
I probably should uncomment this. I'm just concerned about a for-loop setting individual pixels leaving to having thousands upon thousands of registered drawings, but I don't think we're doing that in these cases.